src/dal/Jalasoft.TeamUp.Resumes.DAL/DapperWrapper.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Jalasoft.TeamUp.Resumes.DAL
```

Comment:
we do not need this class since we do not need unit tests for the DAL

Reply:
I delete file.